src/main/kotlin/org/rust/cargo/project/model/CargoProjectService.kt

```diff
@@ -97,11 +97,13 @@ interface CargoProject : UserDataHolderEx {
     val workspaceStatus: UpdateStatus
     val stdlibStatus: UpdateStatus
     val rustcInfoStatus: UpdateStatus
+    val rustupInfoStatus: UpdateStatus
```

Comment:
As I can see, we never update this status it leads to "Project need update" status
![image](https://user-images.githubusercontent.com/2539310/116677000-59270600-a9b0-11eb-80e5-70bdecf7532d.png)
Do you want to use it in this PR? If no, let's drop it